SCHNDR/iocBoot/iocSCHNDR-IOC-01/devices/missing.cmd

```diff
@@ -0,0 +1,3 @@
+##
+msgBox("ERROR: $(IOCNAME) - no device is defined")
```

Comment:
Maybe better with a slightly more verbose msgbox text than just the default "Press OK to continue"

```suggestion
msgBox("ERROR: $(IOCNAME) - no device is defined" "See github.com/isiscomputinggroup/ibex_developers_manual/wiki/schneider_plc for details of how to configure")
```